Scripts/parameters/tour_generation.py

```diff
@@ -61,67 +61,67 @@
 
 tour_generation = {
     "hb_work": {
-        "age_7_17": 0.0059,
-        "age_18_29": 0.2645,
-        "age_30_49": 0.3549,
-        "age_50_64": 0.2785,
-        "age_65_99": 0.0143
+        "age_7_17": 0.0062,
+        "age_18_29": 0.2677,
+        "age_30_49": 0.3645,
+        "age_50_64": 0.2926,
+        "age_65_99": 0.0156
     },
     "hb_edu_basic": {
-        "age_7_17": 0.3254,
-        "age_18_29": 0,
-        "age_30_49": 0,
+        "age_7_17": 0.3804,
+        "age_18_29": 0.0026,
+        "age_30_49": 0.0047,
         "age_50_64": 0,
         "age_65_99": 0
     },
     "hb_edu_upsec": {
-        "age_7_17": 0.0296,
-        "age_18_29": 0.0513,
-        "age_30_49": 0.0031,
-        "age_50_64": 0.0011,
+        "age_7_17": 0.0321,
+        "age_18_29": 0.058,
+        "age_30_49": 0.0053,
+        "age_50_64": 0.002,
         "age_65_99": 0
     },
     "hb_edu_higher": {
-        "age_7_17": 0.0,
-        "age_18_29": 0.0,
-        "age_30_49": 0.0,
-        "age_50_64": 0.0,
-        "age_65_99": 0.0
+        "age_7_17": 0.0046,
```

Comment:
These could be set to zero as higher education tours already have logit tour generation model (parameters above).